### DISCUSSION TITLE
Order placement menu: Adapt units/amounts in footer to new unit system

### DISCUSSION THREAD

**Discussion by twothreenine** [foodcoops/foodsoft#1236] [Decisions]
This is outdated and confusing:
![grafik](https://github.com/foodcoops/foodsoft/assets/22921479/9ed8d728-aff0-4d12-8b09-240ba65a0213)

Suggestion: (omit tolerance for case potatoes, as there's no tolerance input anyway):
![grafik](https://github.com/foodcoops/foodsoft/assets/22921479/0b28d8fd-29d3-4657-afb8-7f00c8367d8a)

----------------

In case of detergent -- old:
![grafik](https://github.com/foodcoops/foodsoft/assets/22921479/e6485658-63cc-4839-8834-1a150b57bc93)

Suggestion:
![grafik](https://github.com/foodcoops/foodsoft/assets/22921479/9f1b8429-6f95-43cb-a5d3-d62d18ac8094)

----------------

In case of beer:

![grafik](https://github.com/foodcoops/foodsoft/assets/22921479/e5a6e5be-6955-4a8b-a3ef-9c06900f3e87)

----------------

So, my suggestion is:
1. Rename "Filled units" ("Volle Gebinde") to "Order result" ("Bestellergebnis") - it's the amount that will be ordered from the supplier (in supplier order unit) if the minimum order quantity is reached, so it's not only about filling units anymore.
(this is the most fitting, short label I could think of -- perhaps you know a better one?)
2. Append the supplier order unit to the _order result_ amount
3. Omit "Units: ..." on the left, since it's included in _order result_
4. <s>Rename "Total units" ("Gesamt-Einheiten") to "Total amount" ("Gesamtmenge"), so that it is consistent with the column titles.
Omit "Total tolerance" if tolerance input is omitted anyway for the respective article</s>
Even simpler: `Total order: <amount> + <tolerance> <(x) group order unit>`
(mathematically incorrect with the + and x, but shorter and better readable IMO)
I'd also put _Total order_ above _Order result_, as it is the more logical way.
5. btw, additionally: Omit _missing_ amount in the table ("0") in case tolerance input is omitted anyway, so that there are less confusing numbers on the screen

**Comment by lentschi** [comment]
I agree that this is currently a bit confusing and might require a redesign, but I think we could live with the current fields in a first version and move this to the "Post-merge" milestone. (Then we might be able to make such a redesign with more people involved.)

**Comment by twothreenine** [comment]
I don't have time to work on this in the near future.

Note: If we make a completely new, responsive version of the order placement menu, we might not need to update the old one.

**Comment by lentschi** [comment]
@twothreenine I've partially picked up on your suggestion in a new issue with the `blocks_release` label:
https://github.com/foodcoops/foodsoft/issues/1271

_What's in the issue's requirements:_ Add units to all quantity displays in the footer.
_What's not in the issue requirements:_ The combination of "total units" and "tolerance" into a single row and the renaming/moving of fields.

I'll leave the discussion open, so we may at some point create another issue to address those parts, but I think what's really relevant for the 5.0 release is in the issue, so I'll remove the `block_release` label from this discussion for now.
If you disagree, please explain why, then we might re-add it.
